.github/workflows/testsuite.yml

```diff
@@ -53,6 +53,10 @@ jobs:
   #                       |__/
   # ascii art small font from https://ascii.co.uk/text
 
+  # "sanity_check" tests -Dusethreads without -DDEBUGGING
+  # "linux-i386" tests -Uusethreads without -DDEBUGGING
+  # All other builds test with -DDEBUGGING
```

Comment:
As far as I can see, it's just `linux`, `make install` and `cygwin` that build with DEBUGGING. Was this intended to say "all other _linux_ builds"?

Reply:
Good catch. Fixed in the (partial) rebase.

Thanks